### ISSUE TITLE
lisp-eval seems to muffle conditions / errors

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#2118]
I seem to recall that a while ago lisp-eval was lettting conditions through, but it's not the case anymore.  Am I mistaken?

I believe we should let them through, otherwise all the lisp-eval errors go unnoticed plus it harder to debug anything since the debugger is not invoked from the REPL.

@aartaka Thoughts?

**Comment by aartaka** [comment]
Yes, does make sense. `evaluate` in general should be more flexible there, I suppose. The same problem happens with REPL, thus hindering interactive debugging there.

**Comment by aartaka** [comment]
Ah, now I remember! It used to `echo-warning` at user too often, that's why I disabled it!

Using `log:debug` should be enough there. Propagating conditions is dangerous, as it happens on a cl-webkit thread and can crush it.

**Comment by Ambrevar** [comment]
We should propagate them when *run-from-repl* is T.
